web/traffic/src/Traffic.elm

```diff
@@ -0,0 +1,314 @@
+port module Traffic exposing (main)
```

Comment:
The only major problem I see with Elm is that the maintainability of this tool may drop to near-zero if you ever decide to leave ESL.